pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -150,10 +150,18 @@ func Run(f *clientcmd.Factory, options *ipfailover.IPFailoverConfigCmdOptions, c
 		return err
 	}
 
-	if len(options.ServiceAccount) == 0 {
-		return fmt.Errorf("you must specify a service account for the ipfailover pod with --service-account, it cannot be blank")
+	if options.VRRPIDOffset < 0 || options.VRRPIDOffset > 254 {
+		return fmt.Errorf("The vrrp-id-offset must be in the range 0..254")
```

Comment:
@ramr The vrrp protocol spec limits the valid vid  values to 1..255, 0 is specifically invalid so there are 254 possible valid values. The doc reference is wrong.
For some reason this was coded so the input vrrp-id-offset is 0-254, this is converted to 1..255 internally.